server/models/users.js

```diff
@@ -0,0 +1,28 @@
+var thinky = require('../config/database');
+
+var r = thinky.r;
+var type = thinky.type;
+
+// Create the model
+var User = thinky.createModel("users", {
+  id: type.string(),
+  name: type.string(),
+  email: type.string()
+});
+
+// Define User API Endpoints
+var UserApi = {
+  getAll: function(req, res){
```

Comment:
Why is a DB query sending back a response (and additionally, why does it have access to the req object)? You want to keep your DB/ORM calls separated from your router/controller logic. The router should ideally, on a request, call some DB function with some params (optionally, not required, depends on your query). The DB query should make the query, and return the data requested to the callback, which can then be processed by the controller/router.